workers/unity/Packages/com.improbable.gdk.core/Worker/WorkerConnector.cs

```diff
@@ -10,17 +10,33 @@
 
 namespace Improbable.Gdk.Core
 {
+    /// <summary>
+    ///     A base class to connect workers via Monobehaviours.
+    /// </summary>
     public class WorkerConnector : MonoBehaviour, IDisposable
     {
         private delegate Task<Worker> ConnectionDelegate();
 
+        /// <summary>
+        ///     The number of connection attempts before giving up.
+        /// </summary>
         public int MaxConnectionAttempts = 3;
+
+        /// <summary>
+        ///     Denotes whether to connect using an external IP address.
+        /// </summary>
         public bool UseExternalIp;
 
+        /// <summary>
+        ///     The Worker representation.
```

Comment:
This sounds a little odd. Also possibly mention that this is only safe to call after the worker has connected.